### PR TITLE
Tweak SFT example for Mixtral

### PR DESCRIPTION
**Pull request by lewtun**
I noticed the OpenAssistant dataset produced some quirks in the loss, so have switched to a pre-fomatted version of UltraChat which works well.

### CODE REVIEW

**Review by pcuenca — APPROVED** [review]
Awesome!

File: mixtral.md [hunk 1]
```diff
@@ -203,18 +203,18 @@ Then you can run the script:
 accelerate launch --config_file examples/accelerate_configs/multi_gpu.yaml --num_processes=1 \
 	examples/scripts/sft.py \
 	--model_name mistralai/Mixtral-8x7B-v0.1 \
-	--dataset_name OpenAssistant/oasst_top1_2023-08-25 \
+	--dataset_name trl-lib/ultrachat_200k_chatml \
 	--batch_size 2 \
 	--gradient_accumulation_steps 1 \
 	--learning_rate 2e-4 \
-	--save_steps 20_000 \
+	--save_steps 200_000 \
```
Comment (lewtun) [inline mixtral.md:210]: This is a hack needed because there's some peculiar issue with checkpoint loading and Mixtral I did not figure out in time
